src/Engine/GUI/Workspaces/SplashScreen.hpp

```
@@ -26,8 +26,8 @@ class SplashScreen : public IWorkspace {
 	// We don't need these functions for splash screens
 	inline void loadSimulationConfig(const std::string &configPath) override {};
 	inline void loadWorkspaceConfig(const std::string &configPath) override {};
-	inline void saveSimulationConfig(const std::string &configPath) override {};
-	inline void saveWorkspaceConfig(const std::string &configPath) override {};
+	inline bool saveSimulationConfig(const std::string &configPath, const std::string &data) override {};
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)